plugin.php

```diff
@@ -19,3 +19,7 @@
 
 // Exit if accessed directly.
 defined( 'ABSPATH' ) || exit;
+
+require_once 'vendor/autoload.php';
```

Comment:
We should probably wrap this is a check to be sure that `vendor/autoload.php` is available. I can see this being an issue only when first getting up and running but that's when we want it to work the most :)

Reply:
good call... and also add some hints somewhere that they need to run `composer dump-autoload` anytime a new file is added

Reply:
done in https://github.com/Automattic/newspack-plugin-starter-template/pull/8/commits/a2f0e324711ade1e7b22d1cff9247b26dfd08aa5